stetho/src/main/java/com/facebook/stetho/inspector/database/DefaultDatabaseFilesProvider.java

```diff
@@ -32,6 +34,45 @@ public DefaultDatabaseFilesProvider(Context context) {
     for (String filename : mContext.databaseList()) {
       databaseFiles.add(new File(filename));
     }
+
+    FilenameFilter filenameFilter = new FilenameFilter() {
+      @Override
+      public boolean accept(File dir, String filename) {
+        return filename.endsWith(".sql")
+            || filename.endsWith(".sqlite")
+            || filename.endsWith(".db");
+      }
+    };
+
+    //Add External Databases if Present
+    if(isExternalAvailable()){
+      final File absoluteFile = mContext.getExternalFilesDir(null).getAbsoluteFile();
+      String[] externalDatabases= absoluteFile.list(filenameFilter);
+      for (String db:externalDatabases){
+        databaseFiles.add(new File(absoluteFile,db));
+      }
+    }
     return databaseFiles;
   }
+
+  /**
+   * @return true if External Storage Present
+   */
+  private static boolean isExternalAvailable() {
```

Comment:
This doesn't strictly seem necessary to me.  You can just test for the file path's existence, no?  If it is not there, then `getExternalFilesDir()` would yield a path that didn't exist right?